release-notes/release-notes57.md

```diff
@@ -38,6 +38,9 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Notifications are now shown when the editor is blocked
+
+Notifications opened with the Notifications Api (`NotificatonApi`) will remain visible when the editor's progress state is set to true using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to true. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

Comment:
```suggestion
Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to true using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will be closed while the editor's progress state is set to true.

For information on the editor `setProgress()` API, see [Editor API - `setProgressState`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).

For information on the Notifications API, see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

This needs breaking up a little and we shouldn't mention `NotificationApi` as that's confusing. I did originally work out that part in the brackets was referring to the type, but types aren't documented and isn't something you'd find in a sentence normally anyways.